content/sensu-go/5.0/guides/troubleshooting.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "How to troubleshoot Sensu Go services"
+linkTitle: "Troubleshooting Sensu Go"
+weight: 20
+version: "5.0"
+product: "Sensu Go"
+platformContent: false
+menu: 
+  sensu-go-5.0:
+    parent: guides
+---
+
+## Service Logging
+
+Logs produced by Sensu services -- i.e. sensu-backend and sensu-agent -- are
+often the best place to start when troubleshooting a variety of issues. 
+
+### Log Levels
+
+Each log message is associated with a log level, indicative of the relative severity of the event being
+logged:
+
+| Log Level          | Description |
+|--------------------|------------------------------------------------------------|
+| panic              | 😱 An error caused the service to exit uncleanly           |
+| fatal              | 👋 An error caused the service to exit cleanly (status 0)  |
+| error              | ❗❗ You should definitely look into this                  |
+| warn               | ❗❓ Maybe you should look into this?                       |
+| info               | 👌 Just so you know                                         |
+| debug              | 📈 Verbose output describing lower-level service operations |
+
+These log levels can be configured by specifying the desired log level as the
+value of `log-level` in the service configuration file (e.g. `agent.yml` or
+`backend.yml` configuration files), or as an argument to the `--log-level`
+command line flag:
+
+{{< highlight shell >}}
+sensu-agent start --log-level debug
+{{< /highlight >}}
+
+Changes to log level via configuration file or command line arguments requires
+restarting the service. 
```

Comment:
Please link to where users can find how to do this

Reply:
@apaskulin Do we have existing documentation on managing Sensu Go services? I’ve looked briefly and not found it.

Reply:
https://docs.sensu.io/sensu-go/5.0/reference/agent/#restarting-the-service and https://docs.sensu.io/sensu-go/5.0/reference/backend/#restarting-the-service